src/service/extension-location.js

```diff
@@ -39,18 +39,11 @@ function calculateScriptBaseUrl(location, isLocalDev, isTest) {
  * @param {string} extensionId
  * @param {boolean=} isLocalDev
  * @param {boolean=} isTest
- * @param {boolean=} isUsingCompiledJs
  * @return {string}
  */
 export function calculateExtensionScriptUrl(location, extensionId, isLocalDev,
-    isTest, isUsingCompiledJs) {
+    isTest) {
   const base = calculateScriptBaseUrl(location, isLocalDev, isTest);
```

Comment:
I am proposing  `calculateEntryPointScriptUrl(location, entryPoint, isLocalDev, isTest)` should not take `isLocalDev` and `isTest` as input. And let local server to do the proxy.